test/Dialect/FIRRTL/parse-errors.fir

```diff
@@ -680,3 +680,11 @@ circuit ConstProbe:
 circuit MultiConst:
   module MultiConst:
     output x : const const UInt<1> ; expected-error {{'const' can only be specified once on a type}}
+
+;// -----
+
+circuit PropAssignNonProperty:
+  module PropAssignNonProperty:
+    input x : UInt<1>
+    output y : UInt<1>
+    propassign x, y ; expected-error {{can only propassign property types}}
```

Comment:
Nit: A check of the "cannot propassign non-equivalent types" would be nice.

Reply:
We only have StringType right now so its impossible to hit :'(